README.md

```diff
@@ -12,12 +12,29 @@ For additional installation instructions and more information visit [https://mic
 
 The extension can be installed directly from source, either for the convenience of using git or to test the latest development version. Clone the desired branch with git
 
-<pre>git clone https://github.com/micheleg/dash-to-dock.git</pre>
-or download the branch from github. A simple Makefile is included. Then run
-<pre>make
+### Build Dependencies
+
+To compile the stylesheet you'll need an implementation of SASS. Dash to Dock supports `dart-sass` (`sass`), `sassc`, and `ruby-sass`. Every distro should have at least one of these implementations, we recommend using `dart-sass` (`sass`) or `sassc` over `ruby-sass` as `ruby-sass` is deprecated.
+
+By default, Dash to Dock will attempt to build with `dart-sass`. To change this behavior set the `SASS` environment variable to either `sassc` or `ruby`.
```

Comment:
Wouldn't better to use `sassc` by default, given it's the one that is what upstream shell also wants?

Reply:
`sassc` is now the default in-code, have to update the documentation :grin:

Reply:
Yeah, noticed afterwards, probably better to squash these changes.

Also I'm reverting some of the changes you did, so would probably better to merge them too.
We can probably use a shared branch if you prefer.

Reply:
Are you reverting more than the DashSpacer? I will squash this down into concerns (SCSS, dash 40 changes, etc.). I can pick your DashSpacer replacement (and others, potentially) onto my squash branch, clean it up, and then we could merge? We could then merge follow-up cleanup and fixes.

Reply:
Yeah, one more change to be reverted is https://github.com/micheleg/dash-to-dock/pull/1402/commits/03b02f3aa65bccbf5aedf09f0b5b99b81253a364#diff-864cfed73316b5b8f434f6eda608c47e1d5af71da8ce7f53755ac2e2ae38849eR679-R681

Reply:
And it's friend on tracking it again.

Reply:
Gotcha out of that commit...

The change to `Main.overview.dash.set_height(-1); // reset default dash size` and the `autohide` CSS addition will likely still be needed.

Reply:
So yeah that stays. But the rest is now basically implemented by https://github.com/micheleg/dash-to-dock/pull/1402/commits/845e10efe58b5e812a5631121ad60ff0f1388184 (before by https://github.com/micheleg/dash-to-dock/pull/1402/commits/7e56142a38755efb0d420f5f2505a098c9534282)